package.json

```diff
@@ -37,6 +37,6 @@
   "license": "MIT",
   "name": "go-datastore",
   "releaseCmd": "git commit -a -m \"gx publish $VERSION\"",
-  "version": "3.4.1"
+  "version": "3.5.0"
```

Comment:
Should we bump up the major version? Although admittedly nobody is using the ordering.